2d-gettingstarted-shared-entities.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+title: Getting Started - Adding Trackable Entities to an existing project
+home: true
+---
+
+This sums up the requirements a project has to comply to for **Trackable Entities** to work properly.
+
+## Declaration
+### 1. Server entities
+ - Implement `ITrackable`
+ - Implement `IMergeable`
+
+### 2. Client entities
+ - Inherit from `EntityBase`
+ - All collection reference properties must be an instance of `ChangeTrackingCollection` (CTC).* All navigation properties must internally wrap the navigated property in a singular CTC as demonstrated [here](https://github.com/TrackableEntities/trackable-entities/blob/develop/Samples/VS2013/WebApiSample.Inheritance/SampleWebApi.Inheritance.Shared.Entities/Models/Product.cs#L83)
+
+##Usage
```

Comment:
I think what we need to focus on here is simply how to customize EF code first entities as trackable entities that are shared between client and server.  So you should rename the file: **2d-gettingstarted-cf-dharted-entities.md**.  Stick to this one topic in this file.

The usage of these entities on the **server** is already described in the other Getting Started pages.  The server-side will usage will vary depending on whether you are using Web API, Patterns, or WCF.

The usage of the entities on the **client** will be the same for *all* scenarios, no matter the kind of entities (CF, separate, shared), so I would suggest creating a separate file called: **2e-gettingstarted-client.md**, in which is described how to use the client-side API.  You can pretty much copy this from the **Retrieving Entities** section of the [Web API Getting Started](http://trackableentities.github.io/2a-gettingstarted-webapi.html).  Then we can update the other getting started sections to link to this client-side getting started.